.github/workflows/deployment-staging.yml

```diff
@@ -82,22 +82,22 @@ jobs:
           password: ${{ secrets.SSH_PASSWORD }}
           script: |
             echo "Stop containers on VPS"
-            cd /srv/tarmac/
+            cd /srv/processing/
             docker image prune -fa
-            docker compose --env-file .env.staging -f docker-compose.staging.yml down
+            docker compose -f docker-compose.production.yml down
```

Comment:
Es gibt die docker-compose.staging.yml in diesem PR ja noch. Ist es dann richtig, sie hier nicht zu nutzen?

Reply:
Hab die Datei jetzt geloescht